sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonSquidSensor.java

```diff
@@ -47,41 +43,36 @@
 import org.sonar.api.measures.Metric;
 import org.sonar.api.rule.RuleKey;
 import org.sonar.api.utils.Version;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
 import org.sonar.plugins.python.coverage.PythonCoverageSensor;
 import org.sonar.python.IssueLocation;
-import org.sonar.python.PythonAstScanner;
 import org.sonar.python.PythonCheck;
 import org.sonar.python.PythonCheck.PreciseIssue;
 import org.sonar.python.PythonConfiguration;
-import org.sonar.python.api.PythonMetric;
+import org.sonar.python.PythonVisitorContext;
 import org.sonar.python.checks.CheckList;
 import org.sonar.python.metrics.FileLinesVisitor;
-import org.sonar.squidbridge.AstScanner;
-import org.sonar.squidbridge.SquidAstVisitor;
-import org.sonar.squidbridge.api.SourceCode;
-import org.sonar.squidbridge.api.SourceFile;
-import org.sonar.squidbridge.api.SourceFunction;
-import org.sonar.squidbridge.indexer.QueryByParent;
-import org.sonar.squidbridge.indexer.QueryByType;
+import org.sonar.python.metrics.MetricVisitor;
+import org.sonar.python.parser.PythonParser;
 
 public final class PythonSquidSensor implements Sensor {
```

Comment:
@ivandalbosco Not in this PR. We already change a lot of things, let's do that separately.